docker/start.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+echo "alias updatemetrics='./updatemetrics.sh'" >> ~/.bashrc
```

Comment:
So I'm thinking about this again.
So this updatemetrics.sh script is basically the script that gets automatically triggered every month to update the metrics in the server. Perhaps it shouldn't even be part of this repo. This repo is about running the metric scripts to update and that is done through `python3 -u -m scripts`.. so perhaps the docker's start script should only be doing that. Then, in my monthly update on the server, I simply have to re-run these scripts from the docker container as per your instructions. 

I think this is what was confusing you and you are right. 

However, we still need a way for the shared charts folder between the scripts and the website.. 

Please ask questions as I'm sure I'm not making a lot of sense at the end of the day :-) !

Reply:
that makes sense